unicorn-play/src/main/scala/org/virtuslab/unicorn/SyncRep.scala

```diff
@@ -0,0 +1,27 @@
+package org.virtuslab.unicorn
+
+import slick.driver.JdbcDriver
+import slick.lifted.{ Rep, Query }
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class SyncRep[T](rep: Rep[T], driver: JdbcDriver) {
+
+  import driver.api._
+
+  protected val queryTimeout = Duration.Inf
+
+  private[unicorn] def invokeAction[R, S <: slick.dbio.NoStream, E <: slick.dbio.Effect](action: DBIOAction[R, S, E])(implicit session: Session) = {
```

Comment:
One is in unicorn-core and the other in unicorn-play. That's the same method I will think about removing the duplication